scio-bigquery/src/main/scala/com/spotify/scio/bigquery/types/TypeProvider.scala

```diff
@@ -79,14 +79,25 @@ private[types] object TypeProvider {
     checkMacroEnclosed(c)
 
     val (r, caseClassTree, name) = annottees.map(_.tree) match {
-      case List(q"case class $name(..$fields) { ..$body }") =>
+      case l @ List(q"$mods class $name[..$tparams] $ctorMods(..$fields) extends { ..$earlydefns } with ..$parents { $self => ..$body }") if mods.asInstanceOf[Modifiers].hasFlag(Flag.CASE) =>
```

Comment:
The problem here was that all the other shorter quasiquotes I have tried were not accepting extra annotations as `mods`. Only this full quote gave me the annotations in the `mods`, if there is better way - please let me know.